### PR TITLE
remove adding new items, or deleting a meeting, when the meeting is from the past

### PR DESCRIPTION
**Pull request by ppelayo1**
### :scroll: Description

- Describe the general shape of this PR (new feature? refactor? bug fix? one-line change?)
This affects meetings that have already taken place.
Remove the ability to upload new agenda items, or delete those meetings.

- Describe what changes are being made
Add a style of visibility:hidden to the two links when the meeting is from the past.

- Describe why these changes are being made
https://trello.com/c/MkIbDwww/164-eng-prevent-adding-new-agenda-items-or-deleting-the-agenda-of-past-meetings

- List the use cases and edge cases relevant to this PR

- Describe how errors will be handled. How will we know if this code breaks in production

- Describe any external libraries/dependencies added or removed in this PR

- Describe any security risks are there regarding this change

- Describe how you tested these changes

- Link to relevant external documentation

--------------------------
### :clipboard: Mandatory Checklist

- [x] Example of a checked item (please remove when creating your Pull Request) 

- [ ] Linked to the Github Issues being addressed using the right sidebar :arrow_right:
- [ ] Have you discussed these changes with the project leader(s)?
- [ ] Do all variable and function names communicate what they do?
- [ ] Were all the changes commented and / or documented?
- [ ] Is the PR the right size? (If the PR is too large to review, it might be good to break it up into multiple PRs.)
- [ ] Does all work in progress, temporary, or debugger code have a TODO comment with links to Github issues?
- [ ] *If you changed the user interface, did you add before and after screenshots to below?*

--------------------------
#### Before

image

#### After

image

--------------------------
### :blue_book: Glossary
- PR = Pull Request

### CODE REVIEW

**Review by JMStudiosJoe — APPROVED** [review]

File: frontend/src/components/MeetingListView/MeetingListItemLinks.jsx [hunk 1]
```diff
@@ -20,8 +21,13 @@ function AdminMeetingItemLinks({ meeting }) {
   const [openUploadModal, UploadModal] = useCSVUpload(meeting);
   const [openDeleteModal, DeleteModal] = useDeleteMeeting(meeting);
 
+  const isFutureItem = isFutureTimestamp(meeting.meeting_start_timestamp);
+  const style = {
```
Comment (JMStudiosJoe) [inline frontend/src/components/MeetingListView/MeetingListItemLinks.jsx:25]: curious but why is this as a style attr and not in the scss module?

Comment (ppelayo1) [reply frontend/src/components/MeetingListView/MeetingListItemLinks.jsx:25]: Felt it was simpler this way.

Comment (JMStudiosJoe) [reply frontend/src/components/MeetingListView/MeetingListItemLinks.jsx:25]: ah please add it to the scss module. the reason is style attri overrides the scss if its on there, almost like a " I did it cause i had to " so if you didnt have to then please add it to the scss, much cleaner that way

Comment (ppelayo1) [reply frontend/src/components/MeetingListView/MeetingListItemLinks.jsx]: Modified it to use a class instead. Also put that class in a utility generalStyles.scss file as it is something that be reused.

**Review by JMStudiosJoe — APPROVED** [review]
nice

**Pull request merged**